### PR TITLE
New Blog Post: Adding Sensors to Web Things

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Adding new blog post: Adding Sensors to Web Things

### CODE REVIEW

**Comment by torgo** [comment]
I am editing this PR do not merge.

**Comment by rzr** [comment]
Thx
can you try to update import, because I edited over Dan by inserting inlined videos,
is the importer supporting this too ?

Note this change should be closed after:
https://github.com/SamsungInternet/blog/pull/79

File: content/posts/NaN-NaN-NaN--adding-sensors-to-web-things/index.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+---
+cover: img.jpg
+title: "Adding Sensors to Web Things"
+description: "We’re big fans of connecting the web to the “Internet of Things.” This guest post from Samsung Open Source Group’s Philippe Coval, following up this post from last year from ziran sun on Internet of things and GRPR, gives some details on how we’re been progressing on this over the past year, specifically when it comes to sensors."
+category: undefined
+img: https://miro.medium.com/max/1200/0*N4edp2L2KgMEnOId.png
+author: RzR
```
Comment (rzr) [inline content/posts/NaN-NaN-NaN--adding-sensors-to-web-things/index.md]: Maybe I should update with my name ? or who cares ?

**Comment by rzr** [comment]
For the record it's published on medium:

https://medium.com/samsung-internet-dev/sensors-webthings-bc48ad9963dd

**Comment by rzr** [comment]
Thanks I can see it at:
https://medium.com/samsung-internet-dev/sensors-webthings-bc48ad9963dd#

**Pull request merged**